hudi-common/src/main/java/org/apache/hudi/common/util/SerializationUtils.java

```diff
@@ -126,6 +127,8 @@ public Kryo newKryo() {
       // Register serializers
       kryo.register(Utf8.class, new AvroUtf8Serializer());
       kryo.register(GenericData.Fixed.class, new GenericAvroSerializer<>());
+      kryo.register(IndexedRecord.class, new GenericAvroSerializer<>());
+      kryo.register(GenericData.Record.class, new GenericAvroSerializer<>());
```

Comment:
Wondering if the new entries are intentionally registered here instead of `HoodieCommonKryoRegistrar#registerClasses` since `HoodieSparkKryoRegistrar` calls `HoodieCommonKryoRegistrar#registerClasses`.

Reply:
I can move it to the common class

Reply:
Actually the common class just registers classes with Kryo but does not register any custom implementations so it does not work

Reply:
I see